src/modules/fancyzones/editor/FancyZonesEditor/Utils/FancyZonesEditorIO.cs

```diff
@@ -1070,7 +1067,7 @@ private bool SetLayoutHotkeys(LayoutHotkeysWrapper layoutHotkeys)
             MainWindowSettingsModel.LayoutHotkeys.CleanUp();
             foreach (var wrapper in layoutHotkeys.LayoutHotkeys)
             {
-                MainWindowSettingsModel.LayoutHotkeys.SelectKey(wrapper.Key.ToString(), wrapper.LayoutId);
+                MainWindowSettingsModel.LayoutHotkeys.SelectKey(wrapper.Key.ToString(CultureInfo.InvariantCulture), wrapper.LayoutId);
```

Comment:
Why `InvariantCulture` here?

Reply:
I assumed that the key is not localized. I will use `CultureInfo.CurrentCulture` instead.